Replace the dynamic version with a fixed one, to prevent a version of 0.0.0
Author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/cclib-1.8.1.orig/pyproject.toml b/cclib-1.8.1/pyproject.toml
index d596004..84dfb2a 100644
--- a/cclib-1.8.1.orig/pyproject.toml
+++ b/cclib-1.8.1/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "cclib"
-dynamic = ["version"]
+version = "1.8.1"
 description = "parsers and algorithms for computational chemistry"
 readme = "README.md"
 authors = [
